src/_guides/language/spec.md

```diff
@@ -1,73 +1,33 @@
 ---
 layout: default
 title: "Dart Language Specification"
-description: "The Dart language specification and proposed changes."
+description: "The Dart language specification."
 ---
 
-Download the _Dart Programming Language Specification_ from
-the Ecma website:
-
-* <a href="http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-408.pdf"
-   target="_blank">Dart Programming Language Specification, 4<sup>th</sup> Edition</a>
-
+Use this page to find the formal Dart language specification.
 For a gentler introduction to the Dart language, see the
 [Dart language tour](/guides/language/language-tour).
 
-You can find previous editions of the specification at
-[Standard ECMA-408](http://www.ecma-international.org/publications/standards/Ecma-408-arch.htm).
-
-
-## Dart 2 changes
+## Dart 2
 
 Dart 2 has changed the Dart language in many ways,
 some of which are not backward-compatible.
-For information about changes and implementation status,
-see the following resources:
-
-* [Dart 2 Updates](/dart-2)
-* [Dart Language and Library Newsletters](https://github.com/dart-lang/sdk/tree/master/docs/newsletter#dart-language-and-library-newsletters)
-* [Language specification source file (TeX format)](https://github.com/dart-lang/sdk/blob/master/docs/language/dartLangSpec.tex)
-* [Informal specifications](https://github.com/dart-lang/sdk/tree/master/docs/language/informal)
-
-
-## Changes in the 4<sup>th</sup> edition
-
-The 4<sup>th</sup> edition of the specification has fewer restrictions on mixins.
-The restrictions on mixins up until this edition were:
+For details, see the [Dart 2 page](/dart-2).
 
-* Mixins must inherit directly from Object.
-* Mixins cannot contain super calls.
-* Mixins cannot define constructors.
+The formal language specification is not yet up-to-date.
+For information about Dart 2 changes, see the following resources.
 
-Only the last one, mixins cannot define constructors, remains.
+* Feature specifications in the dart-lang/sdk repo:
+  * [/docs/language/informal](https://github.com/dart-lang/sdk/tree/master/docs/language/informal)
+* Feature specifications in the dart-lang/language repo:
+  * [/accepted](https://github.com/dart-lang/language/tree/master/accepted)
+  * [/working](https://github.com/dart-lang/language/tree/master/working)
+* [Dart language specification source file (LaTeX format)](https://github.com/dart-lang/sdk/blob/master/docs/language/dartLangSpec.tex)
```

Comment:
Do we want to link to a PDF version? Say, from https://github.com/chalin/dart-spec-and-grammar? :)